### DISCUSSION TITLE
Custom key bindings

### DISCUSSION THREAD

**Discussion by suyogmapara** [vscode-neovim/vscode-neovim#1247] [Q&A]
How do I add custom key bindings say for moving around tabs?

I tried adding below to my init.lua, but it didn't have any effect
nnoremap <slient> gn :call VSCodeNotify('vscode-neovim.Tabnext')<CR>

**Comment by theol0403** [comment]
1. `nmap gn <cmd>tabn<CR>`
2. `nmap gn <cmd>call VSCodeNotify('workbench.action.nextEditorInGroup)<cr>`

When you call vscode command, it has to be a real vscode command (look at keybindings editor)
